plugins/navtiming.js

```diff
@@ -391,6 +392,10 @@
 				catch (ignore) {
 					/* empty */
 				}
+
+				if (data["nt_load_end"] > 0) {
+				    this.fullySent = true
```

Comment:
lint doesn't seem to be complaining for some reason, but this line has spaces instead of tabs